extensions/interactions/guppy.html

```diff
@@ -1,24 +1,42 @@
 <link rel="stylesheet" href="/third_party/static/guppy-b5055b/build/guppy-default.min.css">
+<link rel="stylesheet" href="/third_party/static/guppy-b5055b/build/guppy-default-osk.min.css">
 <script>
   // Use the normal default browser eventlistener without passive set to false.
   // TODO(#8833): Use passive listeners when guppy supports it.
   EventTarget.prototype.addEventListener = (
     EventTarget.prototype.addEventListener._original);
 </script>
 <script type="text/javascript" src="/third_party/static/guppy-b5055b/build/guppy.min.js"></script>
+<script type="text/javascript" src="/third_party/static/guppy-b5055b/build/guppy_osk.js"></script>
 
 <script type="text/javascript">
+  Guppy.use_osk(new GuppyOSK({"goto_tab":"arithmetic"}));
   Guppy.init({
     "path": "/third_party/static/guppy-b5055b/build"
   });
+  Guppy.configure("buttons", ["osk", "controls"]);
+  Guppy.configure("empty_content", '\\color{grey}{\\text{\\small{Type a formula here.}}}')
 </script>
 
 <style>
+  .guppy_help {
+    top: 4% !important;
```

Comment:
Try not to use '!important' tags, isn't there any other way to achieve this?

Reply:
These attributes already have values in guppy-default-osk.min.css, this is supposed to override those values. I'm not aware of any other way to do this.

Reply:
Oh, they are in the style attribute itself in those divs. In that case, I think this is fine.

Reply:
Hm, @iamprayush have you tried making the rule more specific to the element so that your CSS applies correctly? You can refer [this](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity), see notes under "Instead of using !important, consider".

Reply:
@kevintab95 these styles are supposed to be applied to these elements only, not sure what you mean by "more specific to the element". Also, in the link you've referenced, it says 
 > Only use !important on page-specific CSS that overrides foreign CSS (from external libraries, like Bootstrap or normalize.css).

This seems to be relevant here since I'm overriding styles from the third party library.

Reply:
Sorry for not being clear, let me try again. What I meant was, can you look into how to make the CSS specificity for your rule greater so that your rule takes higher precedence than that defined in the guppy-default-osk.min.css. The doc I linked describes a bit about this (ie. which CSS selectors lead to increased specificity.)

Regarding overriding third party styles, this is still a bad pattern, we should try to avoid it. That said, if you are unable to find a way to increase the specificity of the CSS like I mentioned above, we can go ahead and use !important here with a note.

Reply:
Thanks for explaining, @kevintab95! The thing is, the styles that I'm overriding have been assigned to the style attribute of the div directly. See [here](https://github.com/daniel3735928559/guppy/blob/f9440220a73685ea9011c285a4af798f381de53a/build/guppy.js#L4311) for ref. AFAICT, there's no way to override these styles without using `!important`.

Reply:
> See here for ref. AFAICT, there's no way to override these styles without using !important.

Ah okay. Yep, we can't override in this case. Thanks for adding a reference. Just add a comment so that this doesn't get removed later on.

Reply:
Done.